app/src/main/AndroidManifest.xml

```diff
@@ -39,14 +39,14 @@
             android:configChanges="orientation|keyboardHidden|screenSize"
             android:screenOrientation="portrait" />
 
-
         <meta-data
             android:name="firebase_crashlytics_collection_enabled"
             android:value="false" />
-
         <meta-data
             android:name="preloaded_fonts"
             android:resource="@array/preloaded_fonts" />
+
+        <activity android:name=".OnboardingActivity"></activity>
```

Comment:
Let's keep `android:configChanges` and `android:screenOrientation` as with other activities.

Reply:
Sure, missed that part as AS done adding that code to manifest :) Fixed.